test/e2e/testconfig/config/run_debug.sh

```
@@ -0,0 +1,37 @@
+  # print the variable to verify it's set
+  echo "Set $key=$value"
+done < "$envFile"
+
+# Execute the core command directly
+# $1 &
+
+# Get process ID of previously ran command
+# pid=$!
+
+# Execute the command (passed as $1 arg)
+echo "Executing: $1, with pid: $pid, with debug enabled at port: $2"
```

Shellcheck (0.10.0)

[warning] pid is referenced but not assigned.

(SC2154)